packages/ilib-po/test/Parser.test.ts

```diff
@@ -18,9 +18,8 @@
  */
 
 
-// @ts-ignore
+// @ts-expect-error -- untyped package
 import { ResourceString } from "ilib-tools-common";
-import { describe, test, expect } from "@jest/globals";
```

Comment:
Without this line, my IDE complains about these being undefined. It doesn't have any jest smarts built into it apparently.